### ISSUE TITLE
can't see content of 'core/arrange/ArrangeBinder.ts' in web-browser

### ISSUE THREAD

**Issue by mikefills** [wsick/Fayde#242]
Per title.
I do have non-min versions:
```
    <script src="lib/nullstone/dist/nullstone.js"></script>
    <script src="lib/minerva/dist/minerva.js"></script>
    <script src="lib/fayde/dist/fayde.js"></script>
```

In IE it gives me this:
![snip_20160204154815](https://cloud.githubusercontent.com/assets/15019084/12816793/22e74ef2-cb57-11e5-969a-8dbb0cdd7562.jpg)

**Comment by mikefills** [comment]
There's definitely a problem with source maps

The source map does not have for some ts files any content, looking at the json object I can see the sourcesContent[29] is null, where 29 correspondes to the 'core/arrange/ArrangeBinder.ts'

I tried to debug the gulp-sourcemaps\index.js and I could see that in the init() the filepath is "arrange\ArrangeBinder.ts" which is wrong, so that's why the content ends up null, it can't get the content of that file path.

I haven't yet dug into how gulp-sourcemaps receives the wrong path from `gulp.src(meta.buildfiles)`

```
var tsResult = gulp.src(meta.buildfiles)
            .pipe(sourcemaps.init({ debug : true }))

```

**Comment by mikefills** [comment]
something is wrong with the slashes

```
            'typings/**/*.d.ts',
            'lib/perfex/dist/perfex.d.ts',
            'src/_Version.ts',
            'src/*.ts',
            'src/pipe/*.ts',
            'src/core/*.ts',
            'src/core/**/*.ts',
            'src/**/*.ts'
```

**Comment by BSick7** [comment]
Great research. Thanks.

**Comment by BSick7** [comment]
In minerva, I added 'src/pipe/*.ts' to make it easier to develop.
This ensures that the compiled code for those files is listed first in the distributable.
minerva is fairly stable so I can revert to using `/// <reference ...` to solve this problem.

My tests are passing locally, but travis is failing, so I need to resolve before merging.

**Comment by BSick7** [comment]
Fixed in minerva v0.7.6.
